src/App.tsx

```diff
@@ -1,16 +1,25 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  const onAddFunction = (movie: Movie) => {
```

Comment:
```suggestion
  const handleMovieAdd = (movie: Movie) => {
```
You don't need to add a word function in the name, also handlers name in components starts with `handle`
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364